contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,505 @@
+---
+title: Identifying users
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+
+---
+Linking events to specific users enables you to gain full insights as to how they're using your product across different sessions, devices, and platforms. 
+
+This is straight forward to do when [capturing backend events](/docs/getting-started/send-events#3-capture-backend-events), as you associate events to a specific user using `distinct_id`, which is a required argument. However, on the [frontend]((/docs/getting-started/send-events#sending-custom-properties-on-an-event)), this is not a required argument and thus events can be submitted anonymously. In order to link events from anonymous to specific users, you need to use `identify`:
+
+<MultiLanguage selector="tabs">
+
+```javascript
+posthog.identify(
+    'distinct_id',
+    { email: 'max@hedgehogmail.com', name: 'Max Hedgehog' } 
+    {}, 
+);
+```
+
+```Android
+PostHog.with(this)
+       .identify(distinctID, new Properties()
+                                .putValue("name", "My Name")
+                                .putValue("email", "user@posthog.com"));
+```
+
+```iOS
+posthog.identify("distinct_id", 
+          properties: ["name": "Max Hedgehog", "email": "max@hedgehogmail.com"])
+```
+
+</MultiLanguage>
+
+## How `identify` works
+
+When a user starts browsing your website or app, they'll be automatically assigned an **anonymous ID**, which is then stored locally. This allows us to track anonymous users – even across different sessions.
+
+> Note that depending on your [persistence configuration](/docs/libraries/js#persistence), the anonymous ID may not be stored, and thus regenerated across sessions.
+
+By calling `identify` with a `distinct_id` of your choice (usually the user's ID in your database, or their email), you are linking the anonymous ID and distinct ID together.
+
+Thus all past and future events made with that anonymous ID are now associated with the distinct ID. This enables you to do things like associate events with a user from before they log in for the first time, or associate their events across different devices or platforms.
+
+> **A note on using `identify` in the backend:**
+>
+> Although it is technically possible to call `identify` using our backend SDKs, it is typically most useful in frontends. This is because there is no concept of anonymous sessions in the backend SDKs.
```

Comment:
```suggestion
> Although it is technically possible to call `identify` using our backend SDKs, it is typically most useful in frontends. This is because there is no concept of anonymous sessions in the backend SDKs. `identify` on the backend is used to update user information.
```

Reply:
I would prefer to avoid saying this, as `capture` is the preferred method to do this.